testhelpers/testhelpers.go

```diff
@@ -567,44 +567,87 @@ func PullImageWithAuth(dockerCli client.CommonAPIClient, ref, registryAuth strin
 }
 
 func CopyFile(t *testing.T, src, dst string) {
-	fi, err := os.Stat(src)
+	t.Helper()
+
+	err := CopyFileE(src, dst)
 	AssertNil(t, err)
+}
+
+func CopyFileE(src, dst string) error {
```

Comment:
Why are `CopyFileE` and `RecursiveCopyE` useful? Are there places where we actually want to return errors?

Reply:
In the new code added as part of this PR, I didn't want the `FolderBuildpack` to need a `*testing.T` as then I wouldn't be able to instantiate them at build time. An alternative is to copy this behavior in that context.